test/javascripts/acceptance/chat-transcript-test.js

```diff
@@ -62,28 +64,36 @@ function generateTranscriptHTML(messageContent, opts) {
   );
 
   if (opts.channel && opts.multiQuote) {
-    const originallySent = I18n.t("chat.quote.original_channel", {
+    let originallySent = I18n.t("chat.quote.original_channel", {
       channel: opts.channel,
       channelLink: `/chat/chat_channels/${encodeURIComponent(
         opts.channel.toLowerCase()
       )}`,
     });
+    if (opts.linkTabIndex) {
+      originallySent = originallySent.replace(">#", tabIndexHTML + ">#");
+    }
     transcript.push(`<div class=\"chat-transcript-meta\">
 ${originallySent}</div>`);
   }
 
+  const dateTimeText = opts.showDateTimeText
+    ? moment
+        .tz(opts.datetime, "Australia/Brisbane")
+        .format(I18n.t("dates.long_no_year"))
```

Comment:
Do we need to `tz` for formatting or will `moment(opts.datetime).format(...)` be good enough?

Reply:
@nattsw We use moment.tz for it within the decorator code so I think the tests should be the same. This would work just as well with any other timezone, I just put my own one, it just needs to have something.